docs/site/guides/rest/rest-api-guides.md

```diff
@@ -0,0 +1,7 @@
+---
+lang: en
+title: 'REST API Guides'
+keywords: LoopBack 4.0, LoopBack 4, Node.js, REST
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/rest-api-guides.html
```

Comment:
Similar comment: 

```suggestion
permalink: /doc/en/lb4/Rest-api-guides.html
```

Reply:
I would like to transition our URL scheme to all-lowercase names and eventually to nested paths (`/reference/reserved-binding-keys.html`) to match the directory layout in `docs/site`. 

I encountered some issues when using nested paths, so I am using `-` instead of `/` for now, and also slightly reordering the order of components for better readability:

```
guides/rest-api.md ->rest-api-guides.html
```

We are already following the all-lowercase slug style in migration guide, see e.g. here:

https://github.com/strongloop/loopback-next/blob/ae6427322451c914ae54f44dbb656981e7fbbb81/docs/site/migration/datasources.md#L7

Let's discuss in #5798

Reply:
Based on discussion with @raymondfeng, we will stick with Sentence-style.html to land my PRs and then normalize URL slugs afterwards, as this part may take a bit longer to reach consensus.